testutils/src/main/java/com/amplifyframework/testutils/HubAccumulator.java

```diff
@@ -44,12 +44,14 @@
     private final long quantity;
     private final CopyOnWriteArrayList<HubEvent<?>> events;
     private final AtomicReference<SubscriptionToken> token;
+    private final CountDownLatch firstItemLatch;
 
     private HubAccumulator(
             @NonNull HubChannel channel, @NonNull HubEventFilter filter, int quantity) {
         this.channel = channel;
         this.filter = filter;
         this.latch = new CountDownLatch(quantity);
+        this.firstItemLatch = new CountDownLatch(1);
```

Comment:
I was hoping we could just reuse the `latch` -- is it possible? I don't think it makes sense to have multiple ones.

Reply:
Oh...you know what? We don't need that. Just need to check if `latch.getCount() == quantity - 1`. I'll change that.

Reply:
This is done.